rskj-core/src/main/java/co/rsk/core/NetworkStateExporter.java

```diff
@@ -57,52 +56,90 @@ public NetworkStateExporter(RepositoryLocator repositoryLocator, Blockchain bloc
     }
 
     public boolean exportStatus(String outputFile) {
+        return exportStatus(outputFile, "",true,true);
+    }
+
+    public void exportAllAccounts(ObjectNode mainNode,RepositorySnapshot frozenRepository,boolean exportStorageKeys,boolean exportCode) {
+        for (RskAddress addr : frozenRepository.getAccountsKeys()) {
+            if (!addr.equals(RskAddress.nullAddress())) {
```

Comment:
this is duplicated logic, I think you can just define something like this 
```java
private void exportAccount(ObjectNode mainNode, RepositorySnapshot frozenRepository, boolean exportStorageKeys, boolean exportCode, RskAddress addr) {
        if (!addr.equals(RskAddress.nullAddress())) {
            mainNode.set(addr.toString(), createAccountNode(mainNode, addr, frozenRepository, exportStorageKeys, exportCode));
        }
    }
```
and then do the `for` iteration in the `exportStatus`, ending up with something like this:
```java
public boolean exportStatus(String outputFile,String accountKey, boolean exportStorageKeys,boolean exportCode) {
        ...
        try(FileWriter fw = new FileWriter(dumpFile.getAbsoluteFile()); BufferedWriter bw = new BufferedWriter(fw)) {
            ...
            if (accountKey.length() == 0) { // exports all the accounts
                for (RskAddress addr : frozenRepository.getAccountsKeys()) {
                    exportAccount(mainNode, frozenRepository, exportStorageKeys, exportCode, addr);
                }
            } else { // exports just the desired account
                RskAddress addr = new RskAddress(accountKey);
                exportAccount(mainNode, frozenRepository, exportStorageKeys, exportCode, addr);
            }
            ...
            return true;
        } catch (Exception e) {
            ...
            return false;
        }
    }
```

Reply:
done!